src/mod/server.mod/servmsg.c

```diff
@@ -22,13 +22,17 @@
 
 #include "../irc.mod/irc.h"
 #include "../channels.mod/channels.h"
+#include "server.h"
 
 static time_t last_ctcp = (time_t) 0L;
 static int count_ctcp = 0;
 static char altnick_char = 0;
+struct cap_list cap = {"", "", ""};
+char capes[64][32] = { 0 };
```

Comment:
{0} seems to be c99 ok. but pls change it anyway to silence the following compiler warning:
clang -fPIC -g -O2 -pipe -Wall -I. -I../../.. -I../../.. -I../../../src/mod  -DHAVE_CONFIG_H -I/usr/include -g3 -DDEBUG -DDEBUG_ASSERT -DDEBUG_MEM -DDEBUG_DNS  -DMAKING_MODS -c .././server.mod/server.c && mv -f server.o ../
In file included from .././server.mod/server.c:128:
./servmsg.c:31:24: warning: suggest braces around initialization of subobject [-Wmissing-braces]
char capes[64][32] = { 0 };
                       ^
                       {}
1 warning generated.